site/assets/js/application.js

```diff
@@ -39,7 +39,7 @@
     })
   }
 
-  document.querySelectorAll('.toast')
+  document.querySelectorAll('.bd-example .toast')
```

Comment:
Any specific reason for this change? Aren't all `.toast`s children of `.bd-example` already?

Reply:
The live toast is not nested within `.bd-example`. This bit autohides everything I believe and doesn't allow the new live toast to be shown otherwise.